### PR TITLE
Print sys Info at the start of the Windows console

### PR DESCRIPTION
**Pull request by Mohamed-94**
Collect and display system information when starting windows console.

### CODE REVIEW

**Comment by gbaned** [comment]
@Mohamed-94 thank you for your contribution, please sign CLA.

**Comment by Mohamed-94** [comment]
@googlebot I signed it!

> Thanks for your pull request. It looks like this may be your first contribution to a Google open source project (if not, look below for help). Before we can look at your pull request, you'll need to sign a Contributor License Agreement (CLA).
> 
>  **Please visit https://cla.developers.google.com/ to sign.**
> 
> Once you've signed (or fixed any issues), please reply here with `@googlebot I signed it!` and we'll verify it.
> 
> #### What to do if you already signed the CLA
> ##### Individual signers
> * It's possible we don't have your GitHub username or you're using a different email address on your commit. Check [your existing CLA data](https://cla.developers.google.com/clas) and verify that your [email is set on your git commits](https://help.github.com/articles/setting-your-email-in-git/).
> 
> ##### Corporate signers
> * Your company has a Point of Contact who decides which employees are authorized to participate. Ask your POC to be added to the group of authorized contributors. If you don't know who your Point of Contact is, direct the Google project maintainer to [go/cla#troubleshoot](http://go/cla#troubleshoot) ([Public version](https://opensource.google/docs/cla/#troubleshoot)).
> * The email used to register you as an authorized contributor must be the email used for the Git commit. Check [your existing CLA data](https://cla.developers.google.com/clas) and verify that your [email is set on your git commits](https://help.github.com/articles/setting-your-email-in-git/).
> * The email used to register you as an authorized contributor must also be [attached to your GitHub account](https://github.com/settings/emails).
> 
>  **Googlers: [Go here](https://goto.google.com/prinfo/https%3A%2F%2Fgithub.com%2Ftensorflow%2Ftensorflow%2Fpull%2F36482) for more info**.

@googlebot I signed it!

File: tensorflow/tools/pip_package/simple_console_for_windows.py [hunk 1]
```diff
@@ -21,8 +21,14 @@
 
 import code
 import sys
+from tensorflow.tools.test import system_info_lib
 
+# print system information..
+def Info(unused_args):
```
Comment (gunan) [inline tensorflow/tools/pip_package/simple_console_for_windows.py]: All the prints when users load TF are usually misleading,  confuses and annoys users.
What would be the exact use case for this?
When exactly would this be shown?

**Comment by gbaned** [comment]
@Mohamed-94 Can you please check reviewer comments and keep us posted. Thanks!

**Comment by tensorflowbutler** [comment]
It has been 15 days with no activity and the `awaiting response` label was assigned. Is this PR still valid? Assigning the `stalled` label. Please comment to reassure me that this is still being worked on.

**Comment by gbaned** [comment]
I'm going to go ahead and close this PR, because it seems to have stalled. If you're still interested in pursing this (and responding to my comments), please feel free to reopen!

**Pull request closed without merging**